### ISSUE TITLE
Fetch all tags in each event

### ISSUE THREAD

**Issue by macortesn** [egxn/valiu-tags#2]
Why do you do a get request of all tags in each event? Don't you think it can be inefficient?

https://github.com/egxn/valiu-tags/blob/5c7067a15ababf8532b1c451a3fc805e11471d18/App.tsx#L44-L46

**Comment by egxn** [comment]
nope, is a way to get the new tags (an updated list) from the server
